vendor/bat-native-ledger/BUILD.gn

```diff
@@ -63,6 +63,7 @@ source_set("headers") {
     "//brave/components/brave_rewards/browser:browser",
     "//brave/components/brave_rewards/browser:testutil",
     "//brave/vendor/brave-ios:rewards_db",
+    "//chrome/browser/ui"
```

Comment:
this is a layering violation, you cannot include `//chrome/browser` in `vendor`

Reply:
This adds `//chrome/browser/ui` to the `//vendor-bat-native-ledger/` visibility list. Let me know if you have any concerns with that.

Reply:
sorry, I thought this was a deps list, it wasn't all showing up in the diff